src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,44 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    body,
+    comments,
+    user,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments.length !== 0
```

Comment:
This condition might not work if in App.jsx file line 17 returns `null`. 
Try to write with `post.comments.length === 0` condition instead.